shell/gpu/gpu_surface_gl.cc

```diff
@@ -37,29 +37,34 @@ bool GPUSurfaceGL::Setup() {
     return false;
   }
 
-  if (!delegate_->GLContextMakeCurrent()) {
-    // Could not make the context current to create the native interface.
-    return false;
-  }
+  context_ = delegate_->GetGrContext();
+  srgb_support_ = delegate_->SupportsSRGB();
+
+  if (!context_) {
```

Comment:
I think this is confusing, but ok as is.

I haven't updated any of the GL backends except Android.  So Android will get the context by the calling GetGrContext().  The other GL backends will still try to create the GrContext right here (and return false if they fail).